shub/config.py

```diff
@@ -16,6 +16,8 @@
                         pwd_hg_version, pwd_git_version, pwd_version)
 
 
+SH_IMAGES_PREFIX = 'images.scrapinghub.com/project/'
+SH_IMAGES_REGISTRY = SH_IMAGES_PREFIX + '{project}'
```

Comment:
I wonder if there is a use case where that could be expected/wanted behavior (e.g. I want to use the same image for two different projects, perhaps because they have different project settings but are otherwise identical)